packages/calcite-design-tokens/src/index.ts

```diff
@@ -17,19 +16,4 @@ export const globalTokens = await createCalciteTokenFiles({
   references: coreTokens.source,
 });
 
-export const componentTokens = await getFiles(resolve(__dirname, "component")).then((components) =>
```

Comment:
Please correct me if I'm wrong, but this seems unrelated to the group changes. If so, we can either revert here or merge #8280 before merging this one.

Reply:
oop thanks for the catch